### PR TITLE
vstart: fix text format

### PR DESCRIPTION
**Pull request by Svelar**
src/vstart.sh: Fix text format.

Signed-off-by: Rongqi Sun <sunrongqi@huawei.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by Svelar** [comment]
@zdover23 Hi again, could you help review this?

File: src/vstart.sh
Comment (zdover23) [inline src/vstart.sh:223]: Colon use in this list of options is inconsistent. There seem to be three different standards in play: (1) a comma separating the option from its explanation or gloss (206: "-d, --debug"), (2) nothing at all separating the option from its explanation or gloss (217: "-k keep old configuration files (default)"), and (3) a colon separating the option from its explanation or gloss (234: "--cache <pool>: enable cache tiering on pool").

I don't mean to minimize the authority of my station or diminish whatever standing documentation as an institution has in the open-source-software world, but I'll have to check with Josh and Neha before I provide approval of this since it's outside the ceph/doc/ directory.

Comment (zdover23) [reply src/vstart.sh:223]: Also, Stapler seems to be causing the make check to fail with a 404.

Comment (zdover23) [reply src/vstart.sh:223]: @jdurgin @neha-ojha: What's going on with this PR that's causing Stapler to participate in the failure of both the ceph API tests and the make check?

Comment (Svelar) [reply src/vstart.sh:223]: Kinda weird. There is really no uniform standard, we could make it better.

Comment (zdover23) [reply src/vstart.sh:223]: @Svelar: The lack of a standard for the format of flags and options isn't going to stop me from approving and merging this, when the required checks have passed. I'm mostly making a note to myself to return to this matter in a later commit and make this list consistent.

Comment (Svelar) [reply src/vstart.sh:223]: jenkins retest this please

Comment (Svelar) [reply src/vstart.sh:223]: > 

Of course, thanks a lot.

**Comment by Svelar** [comment]
jenkins retest this please

File: src/vstart.sh
Comment (Svelar) [reply src/vstart.sh:223]: After retest, all checks have passed.

Comment (Svelar) [reply src/vstart.sh:223]: Any update?

**Review by zdover23 — APPROVED** [review]

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/51123 - Reef backport
https://github.com/ceph/ceph/pull/51124 - Quincy backport

The backport to Pacific triggers a change to src/vstart.sh so vast that I think it unwise at this time to attempt it.

**Pull request merged**
